bonusopdracht.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+window.addEventListener('load', () => {
+  // Bonusopdracht 1
+  const btnPrice = document.querySelector('.btn-price');
+  const btnAmbilight = document.querySelector('.btn-ambilight');
+  const btnSoldout = document.querySelector('.btn-soldout');
+
+  // Bonusopdracht 2
+  const btnPrice2 = document.querySelector('.btn-price-2');
+  const btnAmbilight2 = document.querySelector('.btn-ambilight-2');
+  const btnSoldout2 = document.querySelector('.btn-soldout-2');
+  const output5 = document.querySelector('.output5');
+
+  // Bonusopdrachten
+  // Opdracht 1d naar functie
+  const sortTvs = () => {
+    return inventory.sort((a, b) => a.price - b.price);
+  };
+
+  // Opdracht 1c naar functie
+  const showAmbilightTvs = () => {
+    return inventory.filter((tv) => tv.options.ambiLight === true);
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return inventory.filter(tv => tv.options.ambiLight);
```